src/application/components/operation.css

```diff
@@ -4,14 +4,27 @@
   padding: 2rem 0 1rem 0;
 }
 
+.resource {
+  padding: 0 0 1rem 0;
+}
+
 .content {
   padding: 0 3rem;
 }
 
 .description {
-  padding: 0 0 2rem;
+  padding: 1rem 0;
 }
 
 .request {
   padding: 0 0 2rem;
 }
+
+.link {
+  color: inherit;
+  text-decoration: none;
```

Comment:
When I tested it I still got decoration on the link, probably need to nest to the a tag or something